test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -254,6 +254,7 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReady", func() {
 			gomega.Expect(k8sClient.Create(ctx, prodWl)).Should(gomega.Succeed())
 			ginkgo.By("checking the 'prod' workload is admitted")
 			util.ExpectWorkloadsToHaveQuotaReservation(ctx, k8sClient, prodClusterQ.Name, prodWl)
+			util.ExpectQuotaReservedWorkloadsTotalMetric(prodClusterQ, 1)
```

Comment:
It seems that in all cases the metric for reserved equals the metric for admitted. Please add an assert to a test where they differ, a good place might be `test/integration/controller/admissionchecks/provisioning/provisioning_test.go`.

Reply:
Added assert on test/integration/scheduler/scheduler_test.go.

Reply:
thanks